modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -279,30 +325,38 @@ public function createOrderTableIfNotExist() {
 							$table->float('value');
 							$table->integer('bits');
 							$table->integer('bits_payed');
-							$table->text('flyp_id');
+							$table->string('blockonomics_currency');
 						}
```

Comment:
For existing orders (before upgrade), we should set this a default value as BTC.

Reply:
Added default for existing orders in 5fb998d